setup.py

```diff
@@ -6,15 +6,17 @@
 import sys
 
 file_path = os.path.dirname(os.path.realpath(__file__))
-ganga_python_dir = os.path.join(file_path, 'python')
```

Comment:
Given this is used in multiple places doesn't it make more sense to define it once?